declaration/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+from core.models import Discount, CustomUser, Currency, Country, LocalWarehouse, ProductType
+
+class PackageDeclaration(models.Model):
+
+    PACKAGE_STATUS = [
+        ('FW', 'Foreign Warehouse'),
+        ( 'LW', 'Local Warehouse'),
+    ]
+    tracking_code = models.CharField(max_length=50, unique=True, blank=True)
+    price = models.DecimalField(max_digits=10, decimal_places=2)
+    discounted_price = models.DecimalField(max_digits=10, decimal_places=2)
+    currency = models.ForeignKey(Currency, on_delete=models.CASCADE)
+    status = models.CharField(max_length=20, choices=PACKAGE_STATUS) #if the approach is right?
```

Comment:
No, Status would be better to be a separate model, as they can have different properties in the next tasks.